lib/SIL/Bridging.cpp

```diff
@@ -135,7 +141,8 @@ Type TypeConverter::getLoweredCBridgedType(AbstractionPattern pattern,
       if (clangTy->isSpecificBuiltinType(clang::BuiltinType::UChar))
         return getDarwinBooleanType();
     }
-    if (clangTy || canBridgeBool)
+    if (bridging != Bridgeability::None &&
+        (clangTy || canBridgeBool))
```

Comment:
Does the `bridging` check obsolete `canBridgeBool`?

Reply:
Unfortunately, no, because `canBridgeBool` is specifically there to make sure we bridge `Bool` to `ObjCBool` when exporting our own `@objc` methods.